### PR TITLE
OIE SIW UI polish

### PR DESCRIPTION
**Pull request by sherizada-okta**
## Description:

A collection of smaller OIE SIW UI bug fixes.

1. The line that intersects the SIW beacon is misaligned. The line should be vertically centered with the beacon.
2. Signing in to app interstitial page - need more space between the title and spinner
3. Authenticator set up page - need to lowercase “Authenticators” in the title
4. Enroll Phone Authenticator page - add a period at the end of the description text
5. Registration page - in “already have an account ?” link, remove extra space between “account” and  “?”
6. Identifier page -  change “Sign Up” to “Sign up” - sentence case
7. Authenticator set up page - change description text to: “Authenticators help protect your account by ensuring only you have access.“
8. Authenticator set up page - too much padding above title

## PR Checklist

- [x] Have you verified the basic functionality for this change?

- [ ] Added unit tests?

- [ ] Added e2e tests

- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?

### Screenshot/Video:
Before
Screenshot 2021-04-05 at 1 27 30 PM

After
Screenshot 2021-04-05 at 1 27 38 PM

Before
Screenshot 2021-04-05 at 2 42 13 PM
Screenshot 2021-04-07 at 1 34 29 PM

After
Screenshot 2021-04-05 at 2 42 19 PM
Screenshot 2021-04-07 at 1 34 35 PM

Before
Screenshot 2021-04-07 at 1 57 53 PM
After
Screenshot 2021-04-07 at 1 57 35 PM

### Issue:

- [OKTA-382658](https://oktainc.atlassian.net/browse/OKTA-382658)
- [OKTA-375000](https://oktainc.atlassian.net/browse/OKTA-375000)

### CODE REVIEW

File: src/v2/view-builder/views/SelectAuthenticatorEnrollView.js [hunk 1]
```diff
@@ -7,8 +7,8 @@ const Body = BaseForm.extend({
   },
   subtitle: function() {
     const subtitle = this.options.settings.get('brandName') ?
-      loc('oie.select.authenticators.enroll.subtitle.custom', 'login', [this.options.settings.get('brandName')]):
-      loc('oie.select.authenticators.enroll.subtitle', 'login');
+      loc('oie.select.authenticators.enroll.subtitle.2.custom', 'login', [this.options.settings.get('brandName')]):
```
Comment (haishengwu-okta) [inline src/v2/view-builder/views/SelectAuthenticatorEnrollView.js:10]: why create a new key instead of reusing.

Comment (sherizada-okta) [reply src/v2/view-builder/views/SelectAuthenticatorEnrollView.js:10]: @haishengwu-okta inconsistent translations, https://oktawiki.atlassian.net/wiki/spaces/eng/pages/741293778/i18n+dev+process+From+Oct+2019+onwards#If-translations-already-exist-for-the-key%3A

Comment (haishengwu-okta) [reply src/v2/view-builder/views/SelectAuthenticatorEnrollView.js:10]: Given OIE is not widely adapted, I feel it's fine to change string in place.
The problem mentioned in the wiki is concern that English and translation are not stay in sync until next translation coming back (~1 month). Don't think it's problem for us?
@nikhilvenkatraman-okta  @pradeepdewda-okta any thoughts?

Comment (nikhilvenkatraman-okta) [reply src/v2/view-builder/views/SelectAuthenticatorEnrollView.js:10]: Yeah, we should be able to update values for `oie` strings. 

Also just a note, the key name `oie.select.authenticators.enroll.subtitle.2.custom` doesn't seem ideal. We should try and avoid using numbers in the key names and use some thing more meaningful for new keys as these keys could potentially be used by customers for customization.

Comment (sherizada-okta) [reply src/v2/view-builder/views/SelectAuthenticatorEnrollView.js:10]: Ok I'll update that wiki with this new info

File: assets/sass/okta-sign-in.scss [hunk 2]
```diff
@@ -74,6 +74,15 @@
       }
     }
   }
+  &:not(.no-beacon) {
```
Comment (haishengwu-okta) [inline assets/sass/okta-sign-in.scss:77]: could this be living in `v2/all.scss`?

File: assets/sass/v2/_success-redirect.scss [hunk 3]
```diff
@@ -0,0 +1,4 @@
+.o-form-head {
```
Comment (haishengwu-okta) [inline assets/sass/v2/_success-redirect.scss:1]: would this impact v1?

Comment (sherizada-okta) [reply assets/sass/v2/_success-redirect.scss:1]: I didn't find `success-redirect` class being used in v1

File: assets/sass/okta-sign-in.scss (hunk 2 above)
Comment (sherizada-okta) [reply assets/sass/okta-sign-in.scss:77]: sure

File: assets/sass/v2/_success-redirect.scss (hunk 3 above)
Comment (sherizada-okta) [reply assets/sass/v2/_success-redirect.scss:1]: oh snap, right these aren't namespaced 😬

**Review by haishengwu-okta — APPROVED** [review]

**Review by aarongranick-okta — APPROVED** [review]

**Pull request closed without merging**